.github/workflows/e2e-sock-trigger.yaml

```diff
@@ -0,0 +1,57 @@
+name: E2ESoakTrigger
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+  workflow_run:
+    workflows: [ApprovalComment]
+    types: [completed]
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        type: string
+      region:
+        required: true
+        default: 'us-east-1'
+        type: choice
+        options:
+          - "us-east-1"
+          - "us-west-2"
+permissions:
+    id-token: write # aws-actions/configure-aws-credentials@v4.0.1
+    statuses: write # ./.github/actions/commit-status/start
+jobs:
+  resolve_cluster:
+    runs-on: ubuntu-latest
+    outputs:
+      PREEXISTING: ${{ steps.create_cluster.outputs.PREEXISTING }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.git_ref }}
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@v4.0.1
+      with:
+       role-to-assume: arn:aws:iam::${{ vars.ACCOUNT_ID }}:role/${{ vars.ROLE_NAME }}
+       aws-region: ${{ inputs.region || 'us-east-1' }}
+       role-duration-seconds: 21600
+    - uses: ./.github/actions/e2e/install-eksctl
+      with:
+        version: v0.163.0
+    - id: create_cluster
+      run: |
+        PREEXISTING=$(eksctl get cluster -o json | jq '.[].Name' | grep soak-periodic)
+        echo "Found existing cluster name \"$PREEXISTING\""
+        if [[ $PREEXISTING != '' ]]; then 
+            echo PREEXISTING="$PREEXISTING" >> "$GITHUB_OUTPUT"
+        fi
+  sock:
+    needs: [resolve_cluster]
+    uses: ./.github/workflows/e2e.yaml
+    with:
+      suite: Integration
+      region: ${{ inputs.region || 'us-east-1' }}
+      workflow_trigger: "soak"
+      preexisiting_cluster: ${{needs.resolve_cluster.outputs.PREEXISTING || '' }}
+      cleanup: true
```

Comment:
Do we always want to clean up on soak testing? I don't think so right?

Reply:
We eventually do want to cleanup the soak testing clusters. The check of when we cleanup the cluster is now held in the sweeper logic

Reply:
So what does this cleanup refer to?

Reply:
This cleanup references us to run the cleanup script. The field is required